be/src/exprs/jit/jit_expr.cpp

```diff
@@ -77,15 +85,10 @@ Status JITExpr::prepare_impl(RuntimeState* state, ExprContext* context) {
             LOG(INFO) << "JIT: JIT compile success, time cost: " << elapsed / 1000000.0 << " ms";
```

Comment:
This log should be removed and added to profile or counted in a metrics.

Reply:
we will do that in next PR soon.